### PR TITLE
feat: Add support for fetch streaming to js SDK

### PR DESCRIPTION
**Pull request by sradevski**
CLOSES CC-441

### CODE REVIEW

**Comment by sradevski** [comment]
@olivermrbl this is a very basic version, and we can enhance it as needed going forward.

File: packages/core/js-sdk/src/client.ts [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import qs from "qs"
-import { ClientFetch, Config, FetchArgs, FetchInput, Logger } from "./types"
+import { events } from "fetch-event-stream"
```
Comment (olivermrbl) [inline packages/core/js-sdk/src/client.ts:2]: comment(non-blocking): looks like this package is only a single file, so we could include it ourselves, if we don't want another dep

**Review by olivermrbl — APPROVED** [review]
Neat, looks good to me!

File: packages/core/js-sdk/src/client.ts (hunk 1 above)
Comment (sradevski) [reply packages/core/js-sdk/src/client.ts:2]: Yeah it is, I also thought about copying it over, but I don't see the harm in having it as a dependency TBH so I thought of leaving it as-is. Let me know if you think we should get rid of it, and just copy it over

Comment (sradevski) [reply packages/core/js-sdk/src/client.ts:2]: I'll get this merged as I want to use it in platform already but if you think we should remove the dependency I'll do it in a separate PR

**Pull request merged**